azurelinuxagent/common/conf.py

```diff
@@ -276,6 +276,10 @@ def get_extensions_enabled(conf=__conf__):
     return conf.get_switch("Extensions.Enabled", True)
 
 
+def get_extensions_fast_track_enabled(conf=__conf__):
+    return conf.get_switch("Extensions.FastTrackEnabled", True)
```

Comment:
@narrieta - We do need to sort the use of configuration and exposing it to the public or not.

Reply:
discussed offline. Adding this parameter is OK